packages/loader/driver-utils/src/network.ts

```diff
@@ -46,14 +46,16 @@ export class GenericNetworkError extends LoggingError implements IDriverErrorBas
 export class AuthorizationError extends LoggingError implements IAuthorizationError {
     readonly errorType = DriverErrorType.authorizationError;
     readonly canRetry = false;
+    readonly claims: { tag: string, value: string | undefined }; // See ITaggedTelemetryPropertyType
 
     constructor(
         errorMessage: string,
-        readonly claims: string | undefined,
+        claims: string | undefined,
         readonly tenantId: string | undefined,
```

Comment:
I guess I should probably also tag tenantId.  It's interesting - it's not always the same value as the User_TenantId value added by Office Fluid Logger.

Reply:
tenant id will be OII, it might be okay to log.

Reply:
Leaving it for now then.